packages/springboard/src/App.vue

```diff
@@ -17,23 +17,25 @@
         <component 
           :is="step.component" 
           v-if="step.number === currentStep"
+          @setNextStepStatus="setNextStepStatus"
```

Comment:
Instead of a value, I would prefer this being a function on the step working out of the state.
This way we can have canGoNextStep be a computed and it can be re-calculated automagically when we hit "previous"

Reply:
done